society-of-biblical-literature-1st-edition-fullnote-bibliography.csl

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="chicago">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="chicago" default-locale="en-US">
   <info>
     <title>Society of Biblical Literature 1st edition (full note)</title>
+    <title-short>SBL</title-short>
```

Comment:
Wouldn't it make sense to add the edition? Maybe change it to "SBL (1st edition)"

Reply:
Agree - while the main function here is findability (which this PR provides) it seems weird to have the exact same short title for multiple styles. @denismaier 's suggestion follows what we do for APA